redis/src/cluster_client.rs

```diff
@@ -108,13 +119,34 @@ impl ClusterClientBuilder {
     /// This does not create connections to the Redis Cluster, but only performs some basic checks
     /// on the initial nodes' URLs and passwords/usernames.
     ///
+    /// When the `tls-rustls` feature is enabled and TLS credentials are provided, they are set for
+    /// each cluster connection.
+    ///
     /// # Errors
     ///
     /// Upon failure to parse initial nodes or if the initial nodes have different passwords or
     /// usernames, an error is returned.
     pub fn build(self) -> RedisResult<ClusterClient> {
         let initial_nodes = self.initial_nodes?;
 
+        #[cfg(not(feature = "tls-rustls"))]
+        let tls_params = None;
+
+        #[cfg(feature = "tls-rustls")]
+        let tls_params = {
+            let retrieved_tls_params = self
+                .builder_params
+                .certs
+                .clone()
```

Comment:
Is this clone necessary ?

Reply:
Not strictly necessary, but it's a simpler way to handle `tls_params` extraction only if `tls-rustls` feature is defined. 

I don't seem a lot of perf penalty here since it should happen only 1x per client build.